specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/stable/2020-07-31/timeseriesinsights.json

```diff
@@ -0,0 +1,2517 @@
+            "$ref": "./examples/QueryGetSeriesPage3.json"
+          },
+          "QueryGetSeriesWithTakePage1": {
+            "$ref": "./examples/QueryGetSeriesWithTakePage1.json"
+          },
+          "QueryGetSeriesWithTakePage2": {
+            "$ref": "./examples/QueryGetSeriesWithTakePage2.json"
+          },
+          "ColdStoreQueryGetSeriesPage1": {
+            "$ref": "./examples/ColdStoreQueryGetSeriesPage1.json"
+          },
+          "ColdStoreQueryGetSeriesPage2": {
+            "$ref": "./examples/ColdStoreQueryGetSeriesPage2.json"
+          },
+          "ColdStoreQueryGetSeriesPage3": {
+            "$ref": "./examples/ColdStoreQueryGetSeriesPage3.json"
+          },
+          "WarmStoreQueryGetSeriesPage1": {
+            "$ref": "./examples/WarmStoreQueryGetSeriesPage1.json"
+          },
+          "WarmStoreQueryGetSeriesPage2": {
+            "$ref": "./examples/WarmStoreQueryGetSeriesPage2.json"
+          },
+          "WarmStoreQueryGetSeriesPage3": {
+            "$ref": "./examples/WarmStoreQueryGetSeriesPage3.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/StoreType"
+          },
+          {
+            "$ref": "#/parameters/ContinuationToken"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/QueryRequest"
+            },
+            "description": "Time series query request body."
+          },
+          {
+            "$ref": "#/parameters/ClientRequestId"
+          },
+          {
+            "$ref": "#/parameters/ClientSessionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful query.",
+            "schema": {
+              "$ref": "#/definitions/QueryResultPage"
+            },
+            "headers": {
+              "x-ms-request-id": {
+                "x-ms-client-name": "serverRequestId",
+                "type": "string",
+                "description": "Server-generated request ID. Can be used to contact support to investigate a request."
+              }
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "#/definitions/TsiError"
+            },
+            "headers": {
+              "x-ms-request-id": {
+                "x-ms-client-name": "serverRequestId",
+                "type": "string",
+                "description": "Server-generated request ID. Can be used to contact support to investigate a request."
+              }
+            }
+          }
+        }
+      }
+    },
+    "/timeseries/instances": {
+      "get": {
+        "tags": [
+          "TimeSeriesInstances"
+        ],
+        "operationId": "TimeSeriesInstances_Get",
```

Comment:
Since this returns a collection, the operation name should be `List` instead of `Get`.

Reply:
Fixed